src/apps/logging/logging.cpp

```diff
@@ -15,9 +15,8 @@
 
 using namespace std;
 using namespace nlohmann;
-using namespace ccf;
 
-namespace ccfapp
+namespace logging
```

Comment:
API design point that we've been a bit lax on: Only `get_rpc_handler` needs to be in namespace ccfapp, the rest of the app can (and probably should) be in its own, clearly-not-CCF namespace. I've moved logging since its our main sample, and removed the `using namespace ccf` so we're clear on the framework types/dependencies.

Reply:
Can we make this loggingapp or app::logging or something less likely to evoke logging.h?

Reply:
Good point - I've gone for `loggingapp` for now.